docs/dom-testing-library/api-queries.md

```diff
@@ -178,6 +179,9 @@ const inputNode = getByLabelText(container, 'Username', {
 })
 ```
 
+Otherwise `byRole('textbox', { name: 'Username' })` works as well and is robust
```

Comment:
Maybe move this up above the example and recommend it first. Also, use an actual query (not just `byRole` because that doesn't exist). Maybe `getByRole`?

Reply:
Done